stylelint-polaris/index.js

```diff
@@ -43,5 +50,34 @@ module.exports = {
       },
       {severity: 'warning'},
     ],
+    [mediaQueriesAllowedList]: [
+      {
+        // Allowed media types and media conditions
+        // https://www.w3.org/TR/mediaqueries-5/#media
+        allowedMedia: [
+          'print',
+          'screen',
+          '-ms-high-contrast',
+          'forced-colors',
+          // TODO: Add utility to @shopify/polaris-tokens to getMediaConditionNames
+          '$p-breakpoints-xs-up',
```

Comment:
What if you make it so it accepts a regex, then you wouldn't need to have write out a big list `/^\$p-breakpoints-(xs|sm|md|lg|xl)-(up|down|only)$/`